docs/changelog.md

```diff
@@ -1,5 +1,9 @@
 # TBD
 
+### Changes
+* Upgraded core to 1.59.2
```

Comment:
Will you need to upgrade the Engine also?

Reply:
i ran the script but i don't see any dependence on container-engine-lib in the go.mod or package.json

Reply:
<img width="1223" alt="image" src="https://user-images.githubusercontent.com/6670312/190665701-abc744ce-23d2-44a5-a9f0-0f1c397b5947.png">

no dependency on the engine server either ithink. let me see

Reply:
<img width="1223" alt="image" src="https://user-images.githubusercontent.com/6670312/190666109-460173ea-2a77-46d0-840e-12edc9b6cb5e.png">

Reply:
sorry, I mean the Engine server's SDK, you've just released a new Engine server. Will you need to add this also here?

Reply:
<img width="1223" alt="image" src="https://user-images.githubusercontent.com/6670312/190666268-8c6e5658-89c8-458f-9e04-34130d483926.png">

Reply:
no dependence!

Reply:
oh! i see, my mistake, go ahead!

Reply:
no worries! thanks for being thorough